### PR TITLE
feat: add regex to prefetch localization

### PR DESCRIPTION
**Pull request by sdo-1A**

### CODE REVIEW

File: src/templates/prefetch.mustache [hunk 1]
```diff
@@ -30,7 +30,7 @@
 
   var language = '{LANG}';
   var languageKey = 'LANG';
-  if (language !== `{${languageKey}}`) {
+  if (language !== `{${languageKey}}` && /^([a-z]{2,3})(-([A-Z]{2}|[A-Za-z]{4,5}))?$/.test(language)) {
```
Comment (fpaul-1A) [inline src/templates/prefetch.mustache:33]: why is that needed ?

**Review by fpaul-1A — APPROVED** [review]

**Review by matthieu-crouzet — APPROVED** [review]

**Review by fpaul-1A — APPROVED** [review]

**Review by matthieu-crouzet — APPROVED** [review]
